app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +13,7 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
```

Comment:
I think the theme name should be more generic. Instead of Task1

Reply:
renamed the theme to CocktailDictionary